### ISSUE TITLE
a11y test PhetMenu

### ISSUE THREAD

**Issue by pixelzoom** [phetsims/joist#660]
In https://github.com/phetsims/joist/commit/cec89de56b7cf10cf9a4193d2778926ce0b1120f for https://github.com/phetsims/tasks/issues/1044, I converted PhetMenu to ES6 and arrow functions. There's a lot of a11y functionality in here that I don't know how to test.  @jessegreenberg or @zepumph could you please verify that no regressions were introduced?

**Comment by pixelzoom** [comment]
Labeling as blocks-publication because it would presumably be bad if a11y is broken for the PhET menu.

**Comment by jessegreenberg** [comment]
Everything about the PhET menu is working great for a11y except the for the "Full Screen" button which is throwing this error in Chrome when activated with keyboard but not mouse.
```
FullScreen.js:51 Uncaught (in promise) TypeError: fullscreen error
    at Object.enterFullScreen (FullScreen.js:51)
    at Object.toggleFullScreen (FullScreen.js:75)
    at callback (PhetMenu.js:297)
    at fire (MenuItem.js:115)
    at Object.click (MenuItem.js:144)
    at Input.dispatchToListeners (Input.js:1852)
    at Input.dispatchToTargets (Input.js:1884)
    at Input.dispatchEvent (Input.js:1810)
    at Input.dispatchA11yEvent (Input.js:1040)
    at Input.clickAction.Action.phetioPlayback (Input.js:637)
```

And a warning is printed in the console:
> FullScreen.js:51 Failed to execute 'requestFullscreen' on 'Element': API can only be initiated by a user gesture.

Not sure yet if it is even related to these changes. This is not happening in the last sim published with a11y (https://phet.colorado.edu/sims/html/molecules-and-light/1.5.0/dependencies.json)

**Comment by jessegreenberg** [comment]
With grunt checkout-timestamp I verified that this is a problem since before https://github.com/phetsims/joist/commit/2a8ddcdaa3aac8ed4572466adc60b64633c7f851 (4 days ago) while ES6 conversion was < 1 day ago in https://github.com/phetsims/joist/commit/cec89de56b7cf10cf9a4193d2778926ce0b1120f. 

So this issue can be closed but I will make another one about trying to toggle full screen from keyboard.
